Makefile

```diff
@@ -129,8 +131,8 @@ uninstall: $(KUSTOMIZE) manifests ## Uninstall CRDs from a cluster
 	$< build config/crd | kubectl delete -f -
 
 .PHONY: deploy
-deploy: $(KUSTOMIZE) manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-	cd config/manager && kustomize edit set image controller=${IMG}
+deploy: $(KUSTOMIZE) manifests  ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+	make set-manifest-image-bmo MANIFEST_IMG=$(IMG_NAME) MANIFEST_TAG=$(IMG_TAG)
 	$< build config/default | kubectl apply -f -
```

Comment:
`manifests` calls 
1. `manifests-generate`  (doesn't require kustomize)
2. `manifests-kustomize` (requires kustomize and it already [installs](https://github.com/metal3-io/baremetal-operator/blob/d3bcdd79e5114eef4901d82a73983090aa7679a6/Makefile#L151) it by itself )
I believe we can drop the `$(KUSTOMIZE)` here ?

Reply:
Actually `$<` points to kustomize here so we need it. I think `$<` is used instead of `$(KUSTOMIZE)` to make sure the path is correct, not sure if it is really needed though.

Reply:
true, I missed that. Thanks